sdk/core/azure-core-serializer-avro-apache/src/main/java/module-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.core.serializer.avro.apache {
+    requires transitive com.azure.core;
+    requires transitive org.apache.avro;
```

Comment:
Should this also export some packages to client libraries?

Reply:
Added exports.